models.py

```
@@ -706,14 +747,14 @@ def loop_views(self):
             try:
                 self.bot_params(self.config.load_params())
-                coins_list = Dex().enabled_coins_list
+                coins_list = self.dex.enabled_coins
                 sleep_message(msg, 10)
                 self.balances(coins_list)
                 sleep_message(msg, 10)
                 self.orders(Dex().api.orders)
                 sleep_message(msg, 10)
                 self.swaps_summary(
-                    Dex().api.rpc(dex.api.rpc("my_recent_swaps", {"limit": 1000}).json())
+                    self.dex.api.rpc(dex.api.rpc("my_recent_swaps", {"limit": 1000}).json())
```

Ruff (0.12.2)

Undefined name `dex`

(F821)